### PR TITLE
chore(all):   fix sempahore build

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
If you have changed a component's source code (not stories, specs, or docs), before merging your branch run `yarn changeset`. This will prompt you to:
- indicate if changes were patch/minor/major for each modified package
- enter a release message

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

**Review by ShanAtDiligent — APPROVED** [review]

File: .semaphore/paprika-status-a11y.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ blocks:
       jobs:
         - name: "Start paprika-status script"
           commands:
-            - nvm use
+            - nvm install
```
Comment (mikrotron) [inline .semaphore/paprika-status-a11y.yml:24]: Oh! So this will _install_ the node version in `.nvmrc`?

**Review by mikrotron — APPROVED** [review]
llgtm 120

File: .semaphore/paprika-status-a11y.yml (hunk 1 above)
Comment (allison-c) [reply .semaphore/paprika-status-a11y.yml:24]: yes

Comment (mikrotron) [reply .semaphore/paprika-status-a11y.yml:24]: TIL 😄

**Pull request merged**